app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
```

Comment:
remove code from 1..15 line you don't need this

Reply:
done